src/game.h

```diff
@@ -554,11 +554,10 @@ class game
         bool check_zone( const zone_type_id &type, const tripoint &where ) const;
         /** Checks whether or not there is a zone of particular type nearby */
         bool check_near_zone( const zone_type_id &type, const tripoint &where ) const;
-        bool is_zone_manager_open();
         void zones_manager();
-        void zones_manager_shortcuts( const catacurses::window &w_info );
-        void zones_manager_draw_borders( const catacurses::window &w_border,
-                                         const catacurses::window &w_info_border, const int iInfoHeight, const int width );
+        /** Is Zone manager open or not - changes graphics of some zone tiles */
+        bool zones_manager_open = false;
```

Comment:
I am not sure what is the point of changing private field with getter function into public field, other than to allow the rest of the game code to mess with value that should be changed in one place only.

I am a C# guy though, so maybe exposing some fields instead of making getter function is common is C++ world.

Reply:
My motivation here is to reduce the interface to the game class.  Sometimes it makes sense to use a getter/setter.  In this case a getter/setter doesn't really add anything, imo.

I'm open to other interpretation as well of course.

Reply:
Hey, I am just covering my back, chances that someone change the value are quite low. But, if it happen, it is on you ;-)

Reply:
Definitely not the end of the world.  I understand your concern.  If you feel strongly about it I will change it back.

Reply:
I don't mind that much, really, I wasn't particularly happy with having private field and public getter separated by pages of code either. 

I would go with what is common in this project, but who know what that is given the general state of the code.